mutect_1.1.5/mutect_1.1.5.cwl

```diff
@@ -656,7 +662,7 @@ outputs:
         ${
           if (inputs.out)
             return inputs.out;
-          return null;
+          return 'null';
         }
```

Comment:
Do we need to specify return null?

Reply:
Hi @rhshah I just modified the output file names in my latest commit. Can you please try again? Thank you!